src/cli/Clip.cpp

```diff
@@ -70,28 +77,39 @@ int Clip::executeWithDatabase(QSharedPointer<Database> database, QSharedPointer<
         return EXIT_FAILURE;
     }
 
+    QString selectedAttribute = parser->value(AttributeOption);
```

Comment:
@sjamesr last nitpick:
Should we raise if both `-t` and `-a` are used?
e.g. `clip -t -a some_other_field`

Reply:
Also, will `selectedAttribute` be populated if the `-t` option is used?

Reply:
Added an error message if more than one attribute is requested, or if `--totp` and `-a` are both specified.

Well spotted about `selectedAttribute` not being populated, I added a test for that too.